### ISSUE TITLE
How to uninstall OneDrive?

### ISSUE THREAD

**Issue by sindzicat** [Klocman/Bulk-Crap-Uninstaller#171]
Hello!

How can I uninstall onedrive? I don't see it in uninstaller.

Windows 8.1

Best regards,
Andrey.

**Comment by Klocman** [comment]
Hello!

Is onedrive currently installed/working? BCU should be able to detect it. Make sure you have Settings > Detection > Pre-defined rules enabled.

**Comment by sindzicat** [comment]
> Is onedrive currently installed/woring?

Sorry, I really don't know. I just installed Windows 8.1 yesterday. I don't see onedrive in explorer, I see it only in start menu:

![image](https://user-images.githubusercontent.com/33900895/34954925-5820445e-fa33-11e7-977e-e1f17b63cb10.png)

I can't remove onedrive from start menu and bcu uninstaller doesn't see it.

> Make sure you have Settings > Detection > Pre-defined rules enabled.

Yes, this is enabled.

**Comment by Klocman** [comment]
What happens when you click on this OneDrive shortcut? Maybe it's just a shortcut to an installer and OneDrive isn't actually installed.

Can't you remove this tile? Sorry, I don't have Win 8.1 to test this myself.

**Comment by sindzicat** [comment]
I found that this tile is a shortcut actually. I found it there: `C:\PROGRAMDATA\MICROSOFT\WINDOWS\START MENU\PROGRAMS\FILEMANAGER.LNK`. I moved this shortcut in my own folder and renamed onedrive folder in `%systemroot`. As for this app there is no onedrive in process list. I found that it's not active because I use local account not online one. So it seems for me these steps solved my issue.
Thanks!
